src/models/streaming/AudioConfig.ts

```diff
@@ -0,0 +1,22 @@
+export default class AudioConfig {
+    contentType: string;
+    layout?: string;
+    rate?: number;
+    format?: string;
+    channels?: number;
+    constructor(contentType: string, layout?: string, rate?: number, format?: string, channels?: number){
+        this.contentType = contentType;
+        this.layout = layout;
+        this.rate = rate;
+        this.format = format;
+        this.channels = channels;
+    }
+
+    getContentTypeString (): string {
```

Comment:
you have a whitespace between the `function` and `()`, I don't think this is correct TS standard? 

btw I'm following https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#variable-and-function